### PR TITLE
update gateway docs

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: modules/cli/pages/index.adoc [hunk 1]
```diff
@@ -506,27 +506,48 @@ $ skupper gateway generate-bundle <config-filename> <destination-directory>
...
+
+Running the gateway bundle uses the gateway definition YAML and a certificate to access and expose the service on the {service-network}.
+
+--
+
+. Check the status of the gateway service:
 +
+--
+----
+$ systemctl --user status <gateway-definition-name>
```
Comment (ajssmith) [inline modules/cli/pages/index.adoc:544]: This is to check the gateway that is of type service. For the container types, the equivalent would be:

$ podman inspect <gateway-definition-name>
or 
$ docker inspect <gateway-definition-name>

File: modules/cli/pages/index.adoc [hunk 2]
```diff
@@ -506,27 +506,48 @@ $ skupper gateway generate-bundle <config-filename> <destination-directory>
...
 . From the machine that hosts the service you want to expose:
 +
+--
 ----
 $ mkdir gateway
-$ tar -xvf <gateway-definition-file>
+
+$ tar -xvf <gateway-definition-file> --directory gateway
 $ cd gateway
 $ sh ./launch.py
```
Comment (ajssmith) [inline modules/cli/pages/index.adoc:531]: or alternatively

$ chmod +x launch.py
$ ./launch.py

**Review by ajssmith — APPROVED** [review]
lgtm, thanks!

File: modules/cli/pages/index.adoc (hunk 2 above)
Comment (pwright) [reply modules/cli/pages/index.adoc:531]: I think the `sh` option is better (and answers my question in email)

**Pull request merged**